content/wiki/development/Build.md

```diff
@@ -187,29 +156,26 @@ Here's [Run Inexor](https://github.com/inexorgame/inexor-core/wiki/Run-Inexor).
 ## CMake GUI
 
 The order is:
-1. you run `Conan` in the build dir.
-2. you run `CMake` in the build dir.
-3. you run `make` in the build dir / open the Visual Studio file in your build dir.
+1. you run `CMake` in the build dir.
+2. you run `make` in the build dir / open the IDE specific file.
 
-The second step might allow some better costumisations with the CMake Gui (as you see more parameters)
+The first step might allow some better costumisations with the CMake Gui (as you see all parameters in the overview)
 
    * Select your Inexor root directory for `Where is the source code`
    * Create a new directory within the root directory named `build`
    * Select the new `build` directory for `Where to build the binaries`
    * Click `Configure`
    * Select your desired generator
-     * If you use Visual Studio select VS-Version *Visual Studio 14 2015* and (if you have) the x64-Version so e.g. `Visual Studio 14 2015 Win64`
+     * If you use Visual Studio select VS-Version *Visual Studio 15 2017* and (if you have) the x64-Version so e.g. `Visual Studio 15 2017 Win64`
    * Click `Generate` to generate a project file
 
 
 # macOS
   * if you haven't already install [brew](https://brew.sh)
-  * execute `brew install conan`  
-  * execute `conan remote add inexor https://api.bintray.com/conan/inexorgame/inexor-conan --insert`
-  * execute `brew install nasm`  
+  * execute `brew install conan`
```

Comment:
Installing Conan via Brew has brought up many issues with Conan in the past. And the brew recipe is also not managed by the Conan team.

The Conan team recommends to install Conan via pip on all platforms